electron/src/updater/updateServer.ts

```diff
@@ -0,0 +1,82 @@
+"use strict";
```

Comment:
Here's how Electron AutoUpdater works (this is not my stuff, this is how Electron is implemented):

1. We give it a URL
2. We tell it to check for an update
3. It queries the URL from 1. to check for an update
4. a. If there is no update available, it expects status code 204
    b. If there is an update available, it expects a 200 with the following JSON in the body:
```
        {
            url: "download URL"
        }
```
5. If an update was available, it queries the download URL from the JSON response to download the update in .zip format

There's no way to tell the Electron updater to skip 1., 2. and 3. The thing is, we have our own way of deciding if there's an update or not, by using our release manifest. So, we _already_ know there's an update available. There's absolutely no reason to let Electron AutoUpdater do an unnecessary query to our backend to get the JSON response. Instead, we use this local server to satisfy the query at 3. above, and the only time the AutoUpdater module queries the PXT backend is to download the actual binary.

If there was a way to tell Electron to skip the check for update, we wouldn't need this, but there isn't.